pkgs/stdenv/generic/check-meta.nix

```diff
@@ -64,8 +64,8 @@ let
     in
     if envVar != "" then envVar != "0" else config.allowNonSource or true;
 
-  allowlist = config.allowlistedLicenses or config.whitelistedLicenses or [ ];
-  blocklist = config.blocklistedLicenses or config.blacklistedLicenses or [ ];
+  allowlist = config.allowlistedLicenses;
+  blocklist = config.blocklistedLicenses;
```

Comment:
This is a breaking change, right? Is there any error message for someone who previously set `config.blacklistedLicenses`? I don’t mind removing it with a throw, but it seems like this will silently start permitting licences a user was specifically trying to opt out of.

To be honest I am not sure why release managers were not pinged in advance for a 99‐commit rework PR with many breaking changes merged this late after the freeze…

Reply:
> reintroduce it after branch off

With an `mkRenamedOption` for `{black,white}listedLicenses`

Reply:
In hindsight, I should have used `mkRenamedOption`. I did not do so because they are undocumented attributes.